schemaless/sources.py

```diff
@@ -756,3 +754,5 @@ def yield_records(self):
     AffordableRentalPortfolio.NAME: AffordableRentalPortfolio,
     OEWDPermits.NAME: OEWDPermits
 }
+
```

Comment:
Nit: A small one line comment here on why parcels is separate might be useful for someone looking in here for the first time

Reply:
done